packages/synapse-react-client/src/components/dataaccess/ReviewerDashboard.tsx

```diff
@@ -102,6 +103,13 @@ export function ReviewerDashboard(props: ReviewerDashboardProps) {
               )}
               {hasReviewerPermissions && [
                 <Route exact path="/Submissions" key="/Submissions">
+                  {!hasActPermissions && (
+                    <OrientationBanner
+                      name="DataAccessManagement"
+                      title="Getting Started With Data Access Management"
+                      text="When someone requests access to data, that request will show up here. Clicking on the Request ID will take you to a page where you can review the request."
+                    />
+                  )}
```

Comment:
Show OrientationBanner only for non-ACT reviewers on the “Submissions” tab, as specified [here](https://www.figma.com/file/bALXz8AOxdzQyrsSd4RkVm/Governance-UI%3A-Registration-and-Data-Access?type=design&node-id=2567-11992&t=XtpKoMaYnPxvytt6-4)
<img width="1883" alt="feature_OrientationBanner_DataAccessManagement" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/86b77abc-c997-49ba-adb3-20e70dae18d0">